src/cli.rs

```diff
@@ -86,9 +92,9 @@ pub struct IcebreakersConfig {
 }
 
 impl Config {
-    pub fn from_args(args: Args) -> Self {
+    pub fn from_args(args: Args) -> Result<Self, AppError> {
+        let testgen_hs_path = env::var("TESTGEN_HS_PATH")?;
```

Comment:
So right now, the user that want to run our executable on bare metal (without Docker, Cargo, Nix) has to remember to export `TESTGEN_HS_PATH` themselves.

If I don’t:

```
❯ ./target/debug/blockfrost-platform --network preview --node-socket-path ~/.local/share/blockfrost-platform/preview/node.socket --solitary

Error: Server("environment variable not found")
```

IMO, as a user, I'd prefer that it was found automatically, what do you think?